tests/codebuild/run_all_sample_test.sh

```diff
@@ -11,6 +12,8 @@ crd_namespace=${1}
 run_integration_tests ${crd_namespace}
 run_feature_integration_tests ${crd_namespace}
 verify_integration_tests ${crd_namespace}
+run_update_integration_tests ${crd_namespace}
+verify_update_integration_tests ${crd_namespace}
```

Comment:
Add to china tests, too?

Reply:
my bad, yes

Reply:
We don't seem to be running most tests in China at the moment - feature, delete, namespaced - I have added update tests in any case for now.